docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_effects/index.rst

```diff
@@ -429,7 +429,7 @@ Birth outcomes
 
   This risk outcome pair is not included in GBD.
 
-Hemoglobin level will act as a risk factor for stillbirth, as described in the :ref:`pregnancy model document <other_models_pregnancy>`. For the implementation of this risk effect, hemoglobin risk exposure will be defined as **dichotomous** based on a threshold of 70 grams per liter (severe anemia among pregnant women). Notably, it is assumed that increased risk of stillbirth will result in decreased risk of live birth and vise versa, with no impact on the risk of abortion/miscarriage or ectopic pregnancy.
+Hemoglobin level will act as a risk factor for stillbirth for full term pregnancies (NOTE: not necessary to apply to partial birth pregnancies), as described in the :ref:`pregnancy model document <other_models_pregnancy>`. For the implementation of this risk effect, hemoglobin risk exposure will be defined as **dichotomous** based on a threshold of 70 grams per liter (severe anemia among pregnant women). Notably, it is assumed that increased risk of stillbirth will result in decreased risk of live birth and vise versa, with no impact on the risk of abortion/miscarriage or ectopic pregnancy.
```

Comment:
what does "partial birth" mean in this context?

Reply:
```suggestion
Hemoglobin level will act as a risk factor for stillbirth for full term pregnancies (NOTE: not necessary to apply to partial term pregnancies), as described in the :ref:`pregnancy model document <other_models_pregnancy>`. For the implementation of this risk effect, hemoglobin risk exposure will be defined as **dichotomous** based on a threshold of 70 grams per liter (severe anemia among pregnant women). Notably, it is assumed that increased risk of stillbirth will result in decreased risk of live birth and vise versa, with no impact on the risk of abortion/miscarriage or ectopic pregnancy.
```
?

Reply:
You were right -- I meant "partial term" not "partial birth". Thanks for catching that.